docs/user/os-quick-start.md

```diff
@@ -0,0 +1,194 @@
+# Quick Start Guide for Open-Source Kyma
+
+Follow the steps to get started with the open-source API Gateway module.
+
+## Prerequisites
+
+You have created a k3d cluster and added the Istio and API Gateway modules. See [Quick Install](https://kyma-project.io/#/02-get-started/01-quick-install).
+
+## Context
+This quick start guide shows how to create a sample HTTPBin workload and expose it to the internet using the APIRule custom resource (CR). For this purpose, the guide uses a wildcard public domain `*.local.kyma.dev`.
+
+## Procedure
+
+## Create a Workload
+
+<!-- tabs:start -->
+#### **Kyma Dashboard**
+
+1. In Kyma dashboard, go to **Namespaces** and choose **Create**.
+1. Provide the name `api-gateway-tutorial` and switch the toggle to enable Istio sidecar proxy injection.
+2. Choose **Create**.
+3. In the created namespace, go to **Workloads > Deployemnts** and choose **Create**.
+1. Select the HTTPBin template.
+2. Choose **Create**.
+3. Go to **Configuration > Service Accounts** and choose **Create**. 
+4. Enter `httpbin` as your Service Account's name.
+5. Choose **Create**.
+6. Go to **Discovery and Network > Services** and choose **Create**. 
+7. Provide the following configuration details:
+    - **Name**: `httpbin`
+    - In the `Labels` section, add the following labels:
```

Comment:
My crusade against documentation that describes the UI continues! :D 
```suggestion
    - Add the following labels:
```

Reply:
It's a great idea to omit the sections' names! thanks